storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operations/delete/DeleteByQueryStub.java

```diff
@@ -86,4 +87,17 @@ public void call(DeleteResult deleteResult) {
                 })
                 .checkBehaviorOfObservable();
     }
+
+    void verifyBehavior(@NonNull Single<DeleteResult> deleteResultSingle) {
```

Comment:
It will be nice to use `void verifyBehavior(Observable<DeleteResult>)` just for `deleteResultSingle.toObservable()` 
Here and below